BMM.Core/Utils/LoggingUtils.cs

```diff
@@ -0,0 +1,28 @@
+namespace BMM.Core.Utils;
+
+public static class LoggingUtils
+{
+    public static void AddParameter(this Dictionary<string, object> paramsDictionary,
+        string name,
+        string message,
+        int maxLength)
+    {
+        int partNumber = 1;
+        
+        for (int i = 0; i < message.Length; i += maxLength)
+        {
+            if (maxLength + i > message.Length)
+                maxLength = message.Length - i;
+
+            paramsDictionary.Add(GetName(name, partNumber), message.Substring(i, maxLength));
+            partNumber++;
```

Comment:
Maybe we should limit it to 10 parts. Otherwise AnalyticsId which is added later, might not be sent to the server
https://github.com/MicrosoftDocs/appcenter-docs/issues/969

Reply:
If we had more than 20 parameters, that could explain why we didn't have AnalyticsId on Track played events for a while. Right now Track played has exactly 20 parameters.

Reply:
Also, there are only 200 unique event names per day. Currently we're using 139 unique names. We have to remember to never use variables in the event names.
We might want to change "AppNavigator - Error during RestoreMediaQueue".